HubFramework.xcodeproj/project.pbxproj

```diff
@@ -2222,31 +2232,27 @@
 		8ABD6CAE1DF6EC36005BCB33 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				HUB_NON_ERROR_WARNINGS_0720 = "";
-				HUB_NON_ERROR_WARNINGS_0730 = "$(HUB_NON_ERROR_WARNING_0720) -Wno-error=partial-availability";
 				INFOPLIST_FILE = tests/Info.plist;
 				PRODUCT_BUNDLE_IDENTIFIER = com.spotify.HubFrameworkTests;
 				PRODUCT_NAME = HubFrameworkTests;
 				WARNING_CFLAGS = (
 					"$(inherited)",
-					"$(HUB_NON_ERROR_WARNINGS_$(XCODE_VERSION_MINOR))",
 					"-Wno-documentation-unknown-command",
+					"-Wno-partial-availability",
```

Comment:
Partial availability warnings in test targets don’t strike me as important enough to worry about.